InternetHospital.BusinessLogic/Models/Articles/ArticleCreatingModel.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.FileProviders;
+
+namespace InternetHospital.BusinessLogic.Models.Articles
+{
+    public class ArticleCreatingModel
+    {
+        [Required]
```

Comment:
Why are you using dataannotation instead of fluentannotation?

Reply:
Because when i started doing this task we were still using data annotation.